WalletWasabi/Blockchain/TransactionOutputs/CoinsRegistry.cs

```diff
@@ -11,50 +11,26 @@ namespace WalletWasabi.Blockchain.TransactionOutputs;
 public class CoinsRegistry : ICoinsView
 {
 	/// <remarks>Guarded by <see cref="Lock"/>.</remarks>
-	private HashSet<SmartCoin> Coins { get; } = new();
-
-	/// <remarks>Guarded by <see cref="Lock"/>.</remarks>
-	private HashSet<SmartCoin> LatestCoinsSnapshot { get; set; } = new();
```

Comment:
I don't use snapshots anymore because I make sure to lock every time I update the `Transactions` object. Maybe I also need to make sure to lock every time we modify a coin from outside the `CoinsRegistry`? Anyway if this feel unsafe I can revert this change.